samples/kubeflow/m4dapplication.yaml

```diff
@@ -14,6 +14,7 @@ spec:
     role: Security
   data:
     - dataSetID: "{\"ServerName\":\"cocoMDS3\",\"AssetGuid\":\"ASSET_ID\"}"
+      cataloged: true
```

Comment:
If 'cataloged' were to be replaced with 'catalog_requested', the default false would remove the need to change the existing resources. Kind of backward compatible, in this sense.

Reply:
This flag is just meant to indicate that the manager shouldn't look for the data in the enterprise catalog.  Your approach seems to indicate that the user is requesting that the data be cataloged.  He is not.  The manager's logic is making that decision. For example, if a user is writing to a data store outside of the data lake (future scenario) we do not want to catalog it.